src/Microsoft.FeatureManagement/FeatureDefinition.cs

```diff
@@ -42,5 +42,27 @@ public class FeatureDefinition
         /// A list of variant definitions that specify a configuration to return when assigned.
         /// </summary>
         public IEnumerable<VariantDefinition> Variants { get; set; } = Enumerable.Empty<VariantDefinition>();
+
+        /// <summary>
+        /// A value used to group feature flags.
+        /// A <see cref="Label"/> is used together with a <see cref="Name"/> to uniquely identify a feature.
+        /// </summary>
+        public string Label { get; set; }
+
+        /// <summary>
+        /// An ETag indicating the state of a feature. This value is used to determine whether a feature has changed.
+        /// </summary>
+        public string ETag { get; set; }
+
+        /// <summary>
+        /// A dictionary of tags used to assign additional properties to a feature.
+        /// These can be used to indicate how a feature may be applied.
+        /// </summary>
+        public IReadOnlyDictionary<string, string> Tags { get; set; }
+
+        /// <summary>
+        /// A flag to enable or disable sending telemetry events to the registered <see cref="ITelemetryPublisher">.
+        /// </summary>
+        public bool EnableTelemetry { get; set; }
```

Comment:
`TelemetryEnabled`?

Reply:
Is there a reason for the suggestion? I thought this should be similar to `EnableReplicaDiscovery` we are adding in our .NET configuration provider?

Reply:
Maybe we should change the other one as well.
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-properties

Reply:
@zhenlan

Makes sense. Let's rename it.